src/imperative/imperative_utils.h

```diff
@@ -419,7 +419,7 @@ inline void PushFCompute(const FCompute& fn,
       // mapping from index in input_blobs to index in pre_temp_dst
       std::unordered_map<uint32_t, uint32_t> in_temp_idx_map;
 #if MXNET_USE_MKLDNN == 1
-      InvalidateOutputs(outputs, req);
+      if (exec_type != ExecType::kCrossDeviceCopy) InvalidateOutputs(outputs, req);
```

Comment:
Any other types in here? 
Could you add some comments?

Reply:
Only for `kCrossDeviceCopy`, because `_copy_to` op is implemented with this, and it's quite strange. Not sure if I can explain this clearly in short. I'll try.

Reply:
Comments are added.